ui2/src/pages/page/TopMenu.tsx

```diff
@@ -30,7 +30,7 @@ const TopMenu: React.FC<Props> = ({ title, buttons, activeMenuTitle, buttonDivid
         <div className='top-section'>
             <Menu className='top-menu'>
                 <Menu.Menu position='left'>
-                    <Menu.Item disabled={!activeMenuTitle} onClick={history.goBack}>
+                    <Menu.Item as={!activeMenuTitle ? 'div' : undefined} disabled={!activeMenuTitle} onClick={history.goBack}>
```

Comment:
does this work correctly? if activeMenuTitle is false, display the Item but disable it - otherwise display nothing? 

I could be misinterpreting what `as` is doing here

Reply:
By default, `Menu.Item` renders as a NavLink (eventually, an `<a>` tag) which gets disabled when `!activeMenuTitle`, but is still an `<a>`. 

The issue is that the Welcome Header has `<a>` tags as well, that link to Daml and Daml Hub. So the browser console warns that there are nested `<a>` tags in the DOM, which is invalid HTML

So with `as={!activeMenuTitle ? 'div' : undefined}`, if activeMenuTitle is false then display this menu item as a `<div>` (which in `ui2`, activeMenuTitle is actually always false. I haven't used that prop yet). 

If activeMenuTitle is enabled, the undefined will make it a NavLink by default, which makes sense in this case because it's a link to goBack (we don't want to break the accessibility/browser behavior that comes along with _real_ links vs interactive divs)